shiny/dbsync/app.R

```diff
@@ -1,44 +1,96 @@
 library(shiny)
 library(leaflet)
 library(dbplyr)
-library(rgeolocate)
 library(RPostgreSQL)
 
-# this file can be downloaded from 
-# https://lite.ip2location.com/database/ip-country-region-city-latitude-longitude
```

Comment:
:+1: This means we should also delete the "This site or product includes IP2Location LITE data..." footer, correct?